04 Abstraction Exercises Part 1/emailtextbox-application/src/main/java/com/email/textbox/EmailTextBox.java

```diff
@@ -0,0 +1,24 @@
+package com.email.textbox;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class EmailTextBox {
+    private String value;
+    private boolean enabled;
+    private int length;
+
+    private static final String EMAIL_REGEX = "zandersarmiento@gmail.com";
```

Comment:
this is not a regex

Reply:
i already changed it to regex pattern po sir